[fix] pulsar is running on windows mistake has been fixed (#812)
apache/pulsar-site: docs/deploy-bare-metal-multi-cluster.md

## System requirement

> Currently, Pulsar is available for 64-bit **macOS**, **Linux** and Pulsar is usable through Docker in **Windows**. See [Running Pulsar In Docker](https://pulsar.apache.org/docs/next/getting-started-docker/) for deeper understanding. To use Pulsar, you need to install 64-bit JRE/JDK.

Reviewer: ```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](https://pulsar.apache.org/docs/next/getting-started-docker/) if you want to Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
```
